Sources/PT.PM.Matching/MatchResultDto.cs

```diff
@@ -25,6 +25,7 @@ public MatchResultDto(MatchResult matchResult)
             TextSpan = matchResult.TextSpans.Where(textSpan =>
                 textSpan.Start >= 0 &&
                 textSpan.End <= textSpan.GetSourceFile(sourceFile).Data.Length)
+                .ToList()
                 .Union();
```

Comment:
Not the other way round? :)

If `Union` is LINQ method, I'd call it before `ToList`.

Reply:
No, it's not a LINQ method :)